pkg/hyper/hyper.go

```diff
@@ -92,20 +92,20 @@ func (h *Runtime) Version() (string, string, string, error) {
 // Status returns the status of the runtime.
 func (h *Runtime) Status() (*kubeapi.RuntimeStatus, error) {
 	runtimeReady := &kubeapi.RuntimeCondition{
-		Type:   proto.String(kubeapi.RuntimeReady),
-		Status: proto.Bool(true),
+		Type:   *proto.String(kubeapi.RuntimeReady),
```

Comment:
No need to use `*proto.String()`, just use `kubeapi.RuntimeReady` directly here.

Same with other places.